
We must not use MOZILLA_FIVE_HOME ... if we want a working nvu ...

  - Rene Rebe <rene@exactcode.de>

--- mozilla/composer/app/mozilla.in.vanilla	2005-03-23 10:25:27.000000000 +0100
+++ mozilla/composer/app/mozilla.in	2005-03-23 10:25:52.000000000 +0100
@@ -36,9 +36,7 @@
 MRE_HOME=//opt/nvu/lib/mre/mre-0.90
 
 # honor MOZILLA_FIVE_HOME if it's there
-if [ -n "$MOZILLA_FIVE_HOME" ] ; then
-  dist_bin="$MOZILLA_FIVE_HOME"
-else
+
   # Use run-mozilla.sh in the current dir if it exists
   # If not, then start resolving symlinks until we find run-mozilla.sh
   found=0
@@ -77,7 +75,6 @@
       exit 1
     fi
   fi
-fi
 
 script_args=""
 moreargs=""
